### ISSUE TITLE
vale only lints first file?

### ISSUE THREAD

**Issue by swsnr** [vale-cli/vale#46]
When passing multiple files to `vale`, as in

```console
$ vale --output JSON --no-exit foo.md bar.md
```

`vale` seems to lint only the first file.  I see errors in `foo.md` but no errors from `bar.md` (regardless of the `--output` option), although I'm sure `bar.md` has errors (which do appear if I lint `bar.md` specifically).

Is this expected behaviour?  The `--help` output suggests that I should be able to pass multiple arguments:

```
NAME:
   vale - A command-line linter for prose.

USAGE:
   vale [global options] command [command options] [arguments...]
```

**Comment by jdkato** [comment]
No, this is isn't expected behavior (although, admittedly, it's more of an oversight than an implementation bug). I suppose I should add support for the following use cases:

- `vale file1 file2 ...`
- `vale dir1 dir2 ...`
- `vale dir1 file1 ...`

Thanks for the report!

**Comment by jdkato** [comment]
I've pushed a fix in d8ce6cb1ca6753d035ee716d09a817111b7d1496. After I test it a little more, I'll release v0.7.2.

**Comment by jdkato** [comment]
This should be fixed in [v0.7.2](https://github.com/ValeLint/vale/releases/tag/0.7.2).

**Comment by swsnr** [comment]
Thank you very much for fixing this issue so quickly ❤️
